include/CppUTest/CppUTestConfig.h

```diff
@@ -199,6 +199,27 @@
 #define CPPUTEST_CHAR_BIT 8
 #endif
 
+/* Support for "long long" type */
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
I've added a new commit (separate, so it's easily readable) that allows the library to be built with `USE_LONG_LONG` and successfully link test applications both with and without it (though of course trying to use the long-long APIs from an app that doesn't enable them won't work).

I haven't added tests for them yet (because that's a bit tricky) so coverage will go down a bit, but I'm mostly just pushing it at this point to get your thoughts on the general concept and implementation.  I can remove it again if you're happier with the baked-in-packaging approach (at least on Linux).